components/butterfly/src/server/inbound.rs

```diff
@@ -202,7 +206,7 @@ fn process_ack_mlw(server: &Server,
     match tx_outbound.send((addr, msg)) {
         Ok(()) => {
             for membership in memberships {
-                server.insert_member_from_rumor_mlw(membership.member, membership.health);
+                server.insert_member_from_rumor_mlw_smw(membership.member, membership.health);
```

Comment:
The function we're in (`process_ack_mlw`) needs to have the `_smw` noted in its name and doc comment (and similarly with its callers).

Reply:
Thanks for catching this.  I'll get it updated.